source/transfer_list.rst

```diff
@@ -902,5 +902,54 @@ This manifest contains the SPMC attribute node consumed by the SPMD
      - hdr_size
      - Holds a SPMC manifest image in DT format.
 
+
+**Executable entry point information entry layout (XFERLIST_EXEC_EP_INFO)**
+
+This entry type holds an `entry_point_info` structure.
```

Comment:
As Simon said in an earlier comment, it's important to specify that this refers to the AArch64 variant of entry_point_info. You might even want to call this XFERLIST_EXEC_EP_INFO64.